tests/stub/routing/test_routing_v4x3.py

```diff
@@ -1478,7 +1470,7 @@ def work(tx):
     def test_should_successfully_read_from_readable_router_using_tx_function(self):
         # TODO remove this block once all languages work
         if get_driver_name() in ['dotnet']:
-            self.skipTest("needs ROUTE bookmark list support")
+            self.skipTest("Test failing for some reason")
```

Comment:
Alternatively, you could consider moving the skip into the driver's TestKit backend.

Reply:
That is the plan. I'm going to go through and start moving all the 'dotnet' if statements into the blacklist.